src/lib.rs

```diff
@@ -13,6 +13,8 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(not(feature = "std"), feature(alloc, core_intrinsics))]
 
+#![cfg_attr(feature = "const_generics", feature(const_generics))]
```

Comment:
This should probably be `min_const_generics` now. It has a large chance of being stabilized soon.